pkg/storage/storage_benchmark_test.go

```diff
@@ -348,10 +350,11 @@ func (g *generator) RandomString(length int) string {
 }
 
 func (g *generator) RandomMetricsPoint() MetricsPoint {
+	randomfactor += 1e10
 	return MetricsPoint{
 		Timestamp:         time.Now(),
-		CumulativeCpuUsed: g.rand.Uint64(),
-		MemoryUsage:       g.rand.Uint64(),
+		CumulativeCpuUsed: randomfactor + uint64(g.rand.Int63n(1e8)),
+		MemoryUsage:       randomfactor + uint64(g.rand.Int63n(1e8)),
```

Comment:
I think this bug was introduced when CPU was migrated to cumulative metric, we forgot to make generated data cumulative causing metrics to be dropped. This however should not affect Memory as it is not cumulative.

Reply:
I think this is  this problem exactly